app.js

```diff
@@ -50,36 +56,50 @@ function showButton() {
     submitButton.classList.add('show-button');
 }
 
-function startPlaying() {
-    
+function getComputerValue() {  
+    switch(randomNumber) {
+        case 0:
+            appState.computerSelectedItem = paper;
+            break;
+        case 1:
+            appState.computerSelectedItem = rock;
+            break;
+        default:
+            appState.computerSelectedItem = scissors;
+            break;
```

Comment:
in general, after the default statement you don't need to make a break, because there is nothin after that